PoGo.NecroBot.Logic/Inventory.cs

```diff
@@ -83,13 +83,12 @@ public Inventory(Client client, LogicClient logicClient)
                 {
                     var settings = pokemonSettings.Single(x => x.PokemonId == pokemon.Key);
                     var familyCandy = pokemonFamilies.Single(x => settings.FamilyId == x.FamilyId);
-                    if (settings.CandyToEvolve == 0)
-                        continue;
+                    var amountToSkip = _logicClient.Settings.KeepMinDuplicatePokemon;
+                    var amountPossible = familyCandy.Candy / settings.CandyToEvolve;
```

Comment:
Thanks @mijanx
Was a little hasty moving it out. Fixed here: [508](https://github.com/NecronomiconCoding/NecroBot/pull/508)